proxy/src/main/java/org/dragonet/proxy/network/translator/misc/tile/SignBlockEntityTranslator.java

```diff
@@ -4,39 +4,40 @@
 import com.github.steveice10.mc.protocol.data.message.Message;
 import com.github.steveice10.mc.protocol.data.message.MessageStyle;
 import com.github.steveice10.opennbt.tag.builtin.CompoundTag;
+import com.nimbusds.jose.util.Base64;
 import com.nukkitx.nbt.CompoundTagBuilder;
+import io.netty.handler.codec.base64.Base64Encoder;
+import lombok.extern.log4j.Log4j2;
```

Comment:
Remove base64 stuff pls. Also java has built in base64 support

Reply:
done